assets/controllers/form_signalement_front.js

```diff
@@ -643,6 +656,111 @@ class PunaisesFrontSignalementController {
     return canGoNext;
   }
 
+  initStepRecommandation(){
+    let typeLogement = $("input[name='signalement_front[typeLogement]']:checked").val();
+    $("#recapTypeLogement").empty().append(typeLogement.charAt(0).toUpperCase() + typeLogement.slice(1));
+    $("#recapSuperficie").empty().append($("input[name='signalement_front[superficie]']").val());
+    $("#recapAdresse").empty().append($("input[name='signalement_front[adresse]']").val());
+    $("#recapCodePostal").empty().append($("input[name='signalement_front[codePostal]']").val());
+    $("#recapVille").empty().append($("input[name='signalement_front[ville]']").val());
+    let locataire = $("input[name='signalement_front[locataire]']:checked").val();
+    $("#recapLocataire").empty();
+    $("#recapNomProprietaire").empty();
+    $("#recapNomProprietaireContainer").hide();
+    if(locataire == 1){
+      $("#recapLocataire").empty().append("Locataire");
+      $("#recapNomProprietaireContainer").show();
+      $("#recapNomProprietaire").empty().append($("input[name='signalement_front[nomProprietaire]']").val());
+    }else{
+      $("#recapLocataire").empty().append("Propriétaire");
+    }
+    let logementSocial = $("input[name='signalement_front[logementSocial]']:checked").val();
```

Comment:
utiliser plutôt .is(':checked') qui retourne un bool (pareil partout où tu récupères la valeur de :checked)

Reply:
et nommer les bool en préfixant par is :) (isLogementSocial)

Reply:
Le but n'est pas de savoir si un champ est coché mais de connaître la valeur de celui qui est coché.

Reply:
Ah ok ! Au temps pour moi, ne tiens pas compte de ce commentaire, alors :)